{"level":"error","message":"MongoDB Connection Error: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections","stack":"MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:239:13)\n    at NativeConnection.openUri (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\connection.js:1016:34)\n    at Mongoose.connect (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\mongoose.js:446:15)\n    at connectDB (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\config\\database.ts:34:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-16T11:01:51.388Z"}
{"level":"error","message":"MongoDB Connection Error: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections","stack":"MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:239:13)\n    at NativeConnection.openUri (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\connection.js:1016:34)\n    at Mongoose.connect (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\mongoose.js:446:15)\n    at connectDB (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\config\\database.ts:34:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-16T11:26:47.689Z"}
{"level":"error","message":"MongoDB Connection Error: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections","stack":"MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:239:13)\n    at NativeConnection.openUri (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\connection.js:1016:34)\n    at Mongoose.connect (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\mongoose.js:446:15)\n    at connectDB (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\config\\database.ts:34:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-16T11:28:28.065Z"}
{"error":{"ErrorCodes":"Email not verified","isOperational":true,"statusCode":401},"level":"error","message":"[Global Error Handler] Please verify your email address first","method":"POST","path":"/api/v1/login","stack":"Error: Please verify your email address first\n    at AuthController.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:179:15)","timestamp":"2025-03-16T11:31:42.824Z"}
{"error":{"ErrorCodes":"Email not verified","isOperational":true,"statusCode":401},"level":"error","message":"[Global Error Handler] Please verify your email address first","method":"POST","path":"/api/v1/login","stack":"Error: Please verify your email address first\n    at AuthController.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:179:15)","timestamp":"2025-03-16T11:35:03.973Z"}
{"code":"ESOCKET","command":"CONN","level":"error","library":"SSL routines","message":"Email service connection failed: D4580000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n D4580000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n","reason":"wrong version number","stack":"Error: D4580000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n","timestamp":"2025-03-16T12:26:51.853Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T12:26:51.854Z"}
{"code":"EAUTH","command":"AUTH PLAIN","level":"error","message":"Email service connection failed: Invalid login: 535 5.7.8 Authentication failed Invalid login: 535 5.7.8 Authentication failed","response":"535 5.7.8 Authentication failed","responseCode":535,"stack":"Error: Invalid login: 535 5.7.8 Authentication failed\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection._actionAUTHComplete (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1586:34)\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:554:26)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)\n    at TLSSocket.SMTPConnection._onSocketData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:195:44)\n    at TLSSocket.emit (node:events:519:28)\n    at TLSSocket.emit (node:domain:488:12)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)","timestamp":"2025-03-16T12:35:21.895Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T12:35:21.897Z"}
{"error":{"ErrorCodes":"User already exist","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Email already registered","method":"POST","path":"/api/v1/register","stack":"Error: Email already registered\n    at AuthController.register (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:111:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-16T13:42:39.191Z"}
{"code":"EAUTH","command":"API","level":"error","message":"Email service connection failed: Missing credentials for \"PLAIN\" Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:38)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:375:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)","timestamp":"2025-03-16T14:01:13.576Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T14:01:13.579Z"}
{"isOperational":true,"level":"error","message":"Uncaught Exception: Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:91:21\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\mailer\\index.js:230:21\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:283:36\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:24)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:272:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)","statusCode":500,"timestamp":"2025-03-16T14:01:50.052Z"}
{"code":"EAUTH","command":"API","level":"error","message":"Email service connection failed: Missing credentials for \"PLAIN\" Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:38)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:375:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)","timestamp":"2025-03-16T14:15:49.212Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T14:15:49.214Z"}
{"code":"EAUTH","command":"API","level":"error","message":"Email service connection failed: Missing credentials for \"PLAIN\" Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:38)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:375:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)","timestamp":"2025-03-16T14:16:30.894Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T14:16:30.898Z"}
{"code":"EAUTH","command":"API","level":"error","message":"Email service connection failed: Missing credentials for \"PLAIN\" Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:38)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:375:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)","timestamp":"2025-03-16T14:17:29.356Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T14:17:29.359Z"}
{"code":"EAUTH","command":"API","level":"error","message":"Email service connection failed: Missing credentials for \"PLAIN\" Missing credentials for \"PLAIN\"","stack":"Error: Missing credentials for \"PLAIN\"\n    at SMTPConnection._formatError (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:807:19)\n    at SMTPConnection.login (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:452:38)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-transport\\index.js:375:32\n    at SMTPConnection.<anonymous> (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:215:17)\n    at Object.onceWrapper (node:events:633:28)\n    at SMTPConnection.emit (node:events:519:28)\n    at SMTPConnection.emit (node:domain:488:12)\n    at SMTPConnection._actionEHLO (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:1369:14)\n    at SMTPConnection._processResponse (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:991:20)\n    at SMTPConnection._onData (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\nodemailer\\lib\\smtp-connection\\index.js:772:14)","timestamp":"2025-03-16T14:18:09.345Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T14:18:09.348Z"}
{"level":"error","message":"Failed to send verification email: Missing credentials for \"PLAIN\"","timestamp":"2025-03-16T14:18:23.110Z"}
{"code":"ESOCKET","command":"CONN","level":"error","library":"SSL routines","message":"Email service connection failed: C4090000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n C4090000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n","reason":"wrong version number","stack":"Error: C4090000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\\ws\\deps\\openssl\\openssl\\ssl\\record\\ssl3_record.c:355:\n","timestamp":"2025-03-16T16:27:20.409Z"}
{"level":"error","message":"Unhandled Rejection at:","timestamp":"2025-03-16T16:27:20.411Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:293:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T19:51:16.094Z"}
{"level":"error","message":"MongoDB Connection Error: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections","stack":"MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:239:13)\n    at NativeConnection.openUri (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\connection.js:1016:34)\n    at Mongoose.connect (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\mongoose.js:446:15)\n    at connectDB (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\config\\database.ts:31:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T19:54:06.660Z"}
{"level":"error","message":"MongoDB Connection Error: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections","stack":"MongooseError: Can't call `openUri()` on an active connection with different connection strings. Make sure you aren't calling `mongoose.connect()` multiple times. See: https://mongoosejs.com/docs/connections.html#multiple_connections\n    at NativeConnection.createClient (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:239:13)\n    at NativeConnection.openUri (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\connection.js:1016:34)\n    at Mongoose.connect (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\mongoose.js:446:15)\n    at connectDB (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\config\\database.ts:32:20)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T19:56:55.690Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:293:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T19:58:29.529Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:298:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:00:52.773Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:298:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:02:30.616Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:298:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:13:34.343Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:299:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:14:19.467Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:299:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:15:35.243Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:299:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:16:08.138Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:24:26.408Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:27:21.111Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T20:40:39.422Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T21:01:07.702Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T21:02:00.926Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:28:17.141Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:28:42.462Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:34:32.542Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:35:29.981Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:39:55.979Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:46:04.078Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:301:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:49:46.261Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:50:54.418Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:52:53.390Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:58:05.146Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T22:59:31.590Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T23:00:48.683Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T23:01:24.701Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T23:04:09.288Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-18T23:04:38.220Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T17:44:22.027Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T17:46:36.826Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T17:48:04.215Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T17:57:39.771Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T18:11:09.943Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T18:55:26.475Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:300:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-23T18:57:48.071Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:305:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:22:20.343Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:310:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:25:42.877Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:313:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:41:55.286Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:313:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:45:59.908Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:314:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:47:24.348Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:314:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:47:45.818Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:317:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:53:19.938Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:317:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:56:18.462Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:317:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:57:27.501Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:317:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T01:59:59.758Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:317:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:03:43.849Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)","timestamp":"2025-03-24T02:16:27.301Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)","timestamp":"2025-03-24T02:21:30.302Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:131:20)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:338:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:23:09.573Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:131:20)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:338:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:24:11.783Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:131:20)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:338:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:24:36.345Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:131:20)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:338:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:26:24.531Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:131:20)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:338:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:26:52.475Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:142:18)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:313:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:29:32.240Z"}
{"error":{},"level":"error","message":"[Global Error Handler] Cannot read properties of undefined (reading 'token')","method":"POST","path":"/api/v1/verify-email","stack":"TypeError: Cannot read properties of undefined (reading 'token')\n    at AuthController.verifyToken (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:142:18)\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:313:53)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at Route.dispatch (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\route.js:119:3)\n    at Layer.handle [as handle_request] (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express\\lib\\router\\index.js:280:10)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\express-rate-limit\\dist\\index.cjs:752:7","timestamp":"2025-03-24T02:30:17.751Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:33:23.379Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:33:45.119Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:34:07.953Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:35:58.139Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:36:48.808Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:39:54.372Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:343:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:41:14.144Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:343:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:42:07.564Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:343:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:43:37.283Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:44:28.781Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:45:14.973Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:342:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-24T02:46:46.762Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:45:20.131Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:45:35.121Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:45:58.336Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:46:14.834Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:46:35.064Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:47:53.887Z"}
{"error":{"ErrorCodes":"Invalid verification token","isOperational":true,"statusCode":400},"level":"error","message":"[Global Error Handler] Invalid token or email already verified","method":"POST","path":"/api/v1/verify-email","stack":"Error: Invalid token or email already verified\n    at AuthController.verifyEmail (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:341:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:47:58.539Z"}
{"error":{"ErrorCodes":"User already exist","isOperational":true,"statusCode":200},"level":"error","message":"[Global Error Handler] Email already verified","method":"POST","path":"/api/v1/verify-account","stack":"Error: Email already verified\n    at AuthController.verifyAccount (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:399:15)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2025-03-25T18:54:22.305Z"}
{"error":{"isOperational":true,"statusCode":500},"level":"error","message":"[Global Error Handler] new password cannot be the same as the old password","method":"POST","path":"/api/v1/forgot-password","stack":"Error: new password cannot be the same as the old password\n    at AuthController.forgotPassword (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\src\\controllers\\authController.ts:674:13)","timestamp":"2025-03-27T10:43:03.782Z"}
{"error":{"_message":"User validation failed","errors":{"password":{"kind":"minlength","message":"Password must be at least 8 characters","name":"ValidatorError","path":"password","properties":{"message":"Password must be at least 8 characters","minlength":8,"path":"password","type":"minlength","value":"build"},"value":"build"}},"message":"User validation failed: password: Password must be at least 8 characters","name":"ValidationError"},"level":"error","message":"[Global Error Handler] User validation failed: password: Password must be at least 8 characters","method":"POST","path":"/api/v1/forgot-password","stack":"ValidationError: User validation failed: password: Password must be at least 8 characters\n    at model.Document.invalidate (C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\document.js:3334:32)\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\document.js:3095:17\n    at C:\\Users\\FOLTI-ADMIN\\Documents\\Ambitious_data\\VTU\\node_modules\\mongoose\\lib\\schemaType.js:1407:9\n    at processTicksAndRejections (node:internal/process/task_queues:77:11)","timestamp":"2025-03-27T10:43:48.375Z"}
